neurom/core/morphology.py

```diff
@@ -240,23 +271,44 @@ def iter_neurites(obj, mapfun=None, filt=None, neurite_order=NeuriteIter.FileOrd
         >>> mapping = lambda n : len(n.points)
         >>> n_points = [n for n in iter_neurites(pop, mapping, filter)]
     """
-    neurites = ((obj,) if isinstance(obj, Neurite) else
-                obj.neurites if hasattr(obj, 'neurites') else obj)
+    neurites = (
```

Comment:
```suggestion
    neurites = (obj, )
    if isinstance(obj, Neurite):
        neurites = obj.neurites
    elif hasaattr(obj, "neurites"):
        neurites = obj
```
shorter and clearer?

Reply:
I think it should be:
```python
if isinstance(obj, Neurite):
    neurites = (obj,)
elif hasattr(obj, "neurites"):
    neurites = obj.neurites
else:
    neurites = obj
```
right?

Reply:
yeah, whatever is correct :)